libs/novui/src/tokens/semanticColors.tokens.ts

```diff
@@ -203,6 +203,16 @@ export const LEGACY_COLOR_SEMANTIC_TOKENS = defineSemanticTokens.colors({
       type: 'color',
     },
   },
+  suggestion: {
+    surface: {
+      value: { base: '{colors.legacy.BGLight}', _dark: '{colors.legacy.B20}' },
+      type: 'color',
+    },
+    text: {
+      value: { base: '#B743FF', _dark: '#AD74FF' },
+      type: 'color',
+    },
+  },
```

Comment:
@antonjoel82 should I add this color, that is only used here to the color tokens? I wasn't sure, because for every palette there, there are shades for each of them (example: amber.40, amber.60 etc)

Reply:
[Figma](https://www.figma.com/design/ybKRCB2ZO5DqICxjxzZGnW/Echo?node-id=3519-33552&m=dev) for context

Reply:
@ainouzgali I would consider this a "design smell" -- we should ask design to stay within our color palettes or add these colors fully. I commented on the Figma :)